docs/_documentations/importing-existing-projects.md

```diff
@@ -93,30 +93,30 @@ Configure your `pom.xml` file as follows:
   </profile>
   ```
 
-  Then add the required Liberty Maven plugin configuration:
+3. Add the required Liberty Maven plugin configuration:
 
-  2b. Liberty server configuration file, `server.xml`, that is located in the source folder that is referenced in the `pom.xml` file.
+- Liberty server configuration file, `server.xml`, that is located in the source folder that is referenced in the `pom.xml` file.
```

Comment:
Maybe... "Add the `server.xml` Liberty server configuration file that is located..."